index.html

```diff
@@ -6,35 +6,26 @@
 
     <title>LeapChat</title>
 
-    <!-- bower:css -->
-    <link rel="stylesheet" href="static/lib/bootstrap/dist/css/bootstrap.css" />
-    <link rel="stylesheet" href="static/lib/font-awesome/css/font-awesome.css" />
-    <!-- endbower -->
-    <link rel="stylesheet" href="static/css/main.css">
+    <!-- inject:css -->
+    <link rel="stylesheet" href="/build/app-8a34b4f915.css">
+    <!-- endinject -->
+
+    <link href="static/lib/font-awesome/fonts/FontAwesome.otf">
+    <link href="static/lib/font-awesome/fonts/fontawesome-webfont.eot">
+    <link href="static/lib/font-awesome/fonts/fontawesome-webfont.svg">
+    <link href="static/lib/font-awesome/fonts/fontawesome-webfont.ttf">
+    <link href="static/lib/font-awesome/fonts/fontawesome-webfont.woff">
+    <link href="static/lib/font-awesome/fonts/fontawesome-webfont.woff2">
```

Comment:
These are 404s. I'm not sure why, routing works for audio file below. Of course, this also assumes that 'bower install' is run in production environment and these files will be available at the specified path.